index.html

```diff
@@ -0,0 +1,171 @@
+
+      <h2 class="date-paragraph">2021.10.22(FRI) - 16(SAT)
+      </h2>
+      <p class="location-paragraph">@ Centro Nacional de Exposiciones Buenos Aires Argentina
+      </p>
+    </div>
+  </main>
+
+
+  <section class="main-program" id="link-2">
+    <div class="container">
+      <h3 class="desk-program">Main Program</h3>
+      <ul class="list">
+
+
+        <li class="cont">
+
+          <div class="inside">
+            <img class="list-image" src="images/icons8-books-orange.png" alt="">
+            <h5>Book Reading</h5>
+            <p class="inside-p">You're favourite authors reading chapters of their bestselling books
+            </p>
+          </div>
+        </li>
+        <li class="cont">
+
+          <div class="inside">
+            <img class="list-image" src="images/icons8-cinema-screen-64.png" alt="">
+            <h5>Movie watching</h5>
+            <p class="inside-p">A cinema theater playing the best based-on books horror movies
+            </p>
+          </div>
+        </li>
+        <li class="cont">
+
+          <div class="inside">
+            <img class="list-image" src="images/icons8-lecture-50.png" alt="">
+            <h5>Authors lecture</h5>
+            <p class="inside-p">Do you have questions for your favorite authors? This is your chance to ask them!
+            </p>
+          </div>
+        </li>
+        <li class="cont">
+
+          <div class="inside">
+            <img class="list-image" src="images/icons8-photo-camera-64.png" alt="">
+            <h5>Meet and Greet</h5>
+            <p class="inside-p">Bring your camera or phone to take pictures with the authors and talk to them!
+            </p>
+          </div>
+        </li>
+        <li class="cont">
+          <div class="inside">
+            <img class="list-image" src="images/icons8-pen-50.png" alt="">
+            <h5>Book Signing</h5>
+            <p class="inside-p">Bring your favorite books and have them signed by these great authors!
+            </p>
+          </div>
+        </li>
+        <li> <button id="link-3" type="submit" class="btn">Join</button></li>
+
+      </ul>
+    </div>
+  </section>
+
+<h2 class="sp-title">Featured Speakers</h2>
+  <section class="speaker-general" id="link-4">
+    <div class="container" id="speaker-container">
+    </div>
+  </section>
+  <footer class="footer-desk">
+    <div>
+      <h4 class="footer-title">Partners</h4>
+      <ul class="first-footer">
+        <li class="mozilla">Mozilla</li>
+        <li class="google">Google</li>
+        <li class="air">Airbnb</li>
+        <li class="kakao">Daumkakao</li>
+        <li class="naver">Naver</li>
+      </ul>
+
+    </div>
+  </footer>
+
+
+
+
+
+
+  <div class="down-home">
+    <div class="container-down">
+      <p class="down-p11">Events for book fans</p>
+      <p class="down-p22">2010 EBFF all rights reserved</p>
+
+
+
+    </div>
+  </div>
```

Comment:
* Kindly fix the footer section of all pages as it looks different from the original design 
* ORIGINAL 
![image](https://user-images.githubusercontent.com/52497246/137445902-294bbf8c-baf4-4d6b-9bb9-645d029c36b8.png)
---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------* YOURS 
![image](https://user-images.githubusercontent.com/52497246/137445963-c6110d8c-1941-4036-8fab-8dc199bf3714.png)